plugins/modules/vmware_cluster_info.py

```diff
@@ -280,8 +280,8 @@ def gather_cluster_info(self):
                 drs_config = cluster.configurationEx.drsConfig
 
                 # VSAN
-                if hasattr(cluster.configurationEx, 'vsanConfig'):
-                    vsan_config = cluster.configurationEx.vsanConfig
+                if hasattr(cluster.configurationEx, 'vsanConfigInfo'):
```

Comment:
@mariolenz Can you please check the backward compatibility of `vsanConfig` with 6.5?

Reply:
> Can you please check the backward compatibility of `vsanConfig` with 6.5?

According to the documentation, it's `vsanConfigInfo` and not `vsanConfig` in 6.5, too: [ClusterConfigInfoEx 6.5](https://code.vmware.com/apis/196/vsphere/doc/vim.cluster.ConfigInfoEx.html). I think it's been `vsanConfigInfo` since the beginning: [ClusterConfigInfoEx  5.5](https://code.vmware.com/apis/197/vsphere/doc/vim.cluster.ConfigInfoEx.html).

I can't test it, though, because we don't have any vCenter 6.5 anymore.

I think I know where this comes from. It's `vsanConfig` (without "Info") when **configuring** VSAN: [ClusterConfigSpecEx](https://code.vmware.com/apis/197/vsphere/doc/vim.cluster.ConfigSpecEx.html). `vsanConfigInfo` in `ClusterConfigInfoEx` when accessing the current VSAN configuration, `vsanConfig` in `ClusterConfigSpecEx` when setting it.

Reply:
OK. No problem.